### PR TITLE
ServiceUI build step update

### PR DESCRIPTION
**Pull request by simaishi**
Updated as per README in manageiq-ui-service repo.
Built on top of https://github.com/ManageIQ/manageiq-appliance-build/pull/180

~~One remaining issue - the app expects to see 'index.html' in public/ui/service directory.  What currently comes out of the build is dynamically generated name. e.g. index-db4658ee4f.html. cc @AllenBW~~

### CODE REVIEW

**Comment by AllenBW** [comment]
@simaishi pr here: https://github.com/ManageIQ/manageiq-ui-service/pull/285 solves this issue 🤘

**Comment by simaishi** [comment]
Ah great, then no remaining issue 😄   Thanks!

File: kickstarts/partials/post/ui_compile.ks.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-npm install -g gulp bower
+npm install -g gulp bower yarn
```
Comment (AllenBW) [inline kickstarts/partials/post/ui_compile.ks.erb]: and  here!  Long live bower! 👑 (as in not)

File: kickstarts/partials/post/ui_compile.ks.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 pushd /var/www/miq/vmdb
   bower -F --allow-root install
```
Comment (AllenBW) [inline kickstarts/partials/post/ui_compile.ks.erb]: And here as well, though it makes more sense if you read my second comment first. :finnadie:

File: kickstarts/partials/post/ui_compile.ks.erb [hunk 3]
```diff
@@ -7,9 +7,9 @@ pushd /var/www/miq/vmdb
   rake evm:compile_sti_loader
 popd
 
-# Self Service UI
-pushd /opt/manageiq/manageiq-ui-self_service
-  npm install
+# Service UI
+pushd /opt/manageiq/manageiq-ui-service
+  yarn install
   bower -F --allow-root install
```
Comment (AllenBW) [inline kickstarts/partials/post/ui_compile.ks.erb]: we can remove bower here, no longer used for  dep management

File: kickstarts/partials/post/ui_compile.ks.erb (hunk 2 above)
Comment (simaishi) [inline kickstarts/partials/post/ui_compile.ks.erb]: @AllenBW No longer need to run bower in 'vmdb' either?  Just confirming as bin/setup and bin/update are still calling it...

Comment (AllenBW) [inline kickstarts/partials/post/ui_compile.ks.erb]: @simaishi as I understand it, this file is is for all the things dealing with sui, if that is the case, then yep, bower is no longer needed 🤘

Comment (simaishi) [inline kickstarts/partials/post/ui_compile.ks.erb]: The references to bower will be removed from bin/setup and bin/update?  cc @himdel

**Comment by simaishi** [comment]
The steps are up-to-date with instruction in manageiq-ui-service repo.  The SUI build no longer fails with this change, but still can't get to SUI login page.  So some other change is still needed here or in other repos.

Not having this change in master is causing appliance builds to hang.  Removing WIP so this can be merged and at least unblock the builds.

File: kickstarts/partials/post/ui_compile.ks.erb (hunk 2 above)
Comment (himdel) [inline kickstarts/partials/post/ui_compile.ks.erb]: @simaishi I'll take care of removing them once Ops UI is no longer using bower :) For now, they are still needed

**Pull request merged**